### DISCUSSION TITLE
SEQ send HTTP Status code 201 but Opentelemetry Proto Spec Says 200 should be sent.

### DISCUSSION THREAD

**Discussion by 0x44454c** [datalust/seq-tickets#2337] [Help] [answered]
Hi Team,
Recently when I was testing out SEQ Opentelemtry log ingestion from Opentelemetry C++ SDK, found out that SEQ sends HTTP status code 201 upon successful request, however opentelemetry SDK expects a 200 HTTP status code which leads to error on client side. 
Further investigation on the specification of Opentelemetry Spec v1.4.0, says collector must send status code of 200.
Please assist me with the resolution of the problem.

**Comment by nblumhardt** [comment, marked as answer]
Thanks for the heads-up; we haven't previously tested integration with the C++ SDK so missed this point.

At this point either using the OpenTelemetry Collector between Seq and your app, or switching to use the gRPC protocol, should provide a workaround. Let us know if you give either of these a try and still have trouble with it.

We'll add this item to the Seq 2025.1 backlog.

**Reply by 0x44454c** [reply]
Thanks! I will surely let you know when I try the suggested methods.
